src/Plugin.php

```diff
@@ -51,6 +51,11 @@ class Plugin implements
 
     public function activate(Composer $composer, IO\IOInterface $io)
     {
+        // Composer 2 comes with parallel downloads built-in
```

Comment:
So this plugin is kind of useless starting composer v2?

Do you have a link telling about such a feature? May be added to the comment.

Reply:
@soullivaneuh yes, it's in the description of the PR ☝️ - I'm not sure adding a link to the comment is a good idea, it is described in Composer's changelog: https://github.com/composer/composer/blob/master/CHANGELOG.md

Reply:
composer2 is slower than this plugin

Reply:
@podarok then you have to contact @Seldaek and ask hin to remove parallel downloads built-in from Composer 2 ;)

Reply:
No, it's not: https://github.com/hirak/prestissimo/issues/233#issuecomment-716115604